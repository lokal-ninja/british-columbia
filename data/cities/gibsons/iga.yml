id: 4733073448
type: node
latitude: 49.4074049
longitude: -123.5309261
postcode: ""
city: Gibsons
street: "Gibsons Way;Hwy 101;Sunshine Coast Highway"
housenumber: 1100
phone: ""
opening_hours: "Mo-Sa 08:00-21:00; Su 08:00-20:00"
website: ""
