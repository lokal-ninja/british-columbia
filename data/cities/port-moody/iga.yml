id: 1706272096
type: node
latitude: 49.28335
longitude: -122.82699
postcode: "V3H 5C9"
city: Port Moody
street: "Ioco Road"
housenumber: 221
phone: "+1-604-461-3373"
opening_hours: "Mo-Su 08:00-22:00"
website: "https://www.igastoresbc.com/"
