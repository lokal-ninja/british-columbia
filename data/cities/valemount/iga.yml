id: 264006773
type: way
latitude: 52.831919500000005
longitude: -119.2642088
postcode: "V0E 2Z0"
city: Valemount
street: "5th Avenue"
housenumber: 1195
phone: "+1-250-566-4331"
opening_hours: ""
website: "http://www.marketplaceiga.com/our-stores/valemount-iga.aspx"
