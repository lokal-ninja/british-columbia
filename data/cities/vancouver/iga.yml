id: 513053228
type: node
latitude: 49.28008
longitude: -123.11746
postcode: ""
city: Vancouver
street: "Robson Street"
housenumber: 489
phone: ""
opening_hours: "Mo-Su 08:00-23:00"
website: ""
