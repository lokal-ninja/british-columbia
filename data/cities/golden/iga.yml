id: 516881665
type: way
latitude: 51.3028
longitude: -116.96852
postcode: "V0A 1H0"
city: Golden
street: "9 Avenue North"
housenumber: 624
phone: ""
opening_hours: "Mo-Su 08:00-22:00"
website: ""
